.storybook/preview.ts

```diff
@@ -9,7 +11,29 @@ const preview: Preview = {
         date: /Date$/i,
       },
     },
+    viewport: {
+      viewports: {
+        ...MINIMAL_VIEWPORTS,
+        desktop: {
+          name: 'Desktop',
+          styles: {
+            width: '1440px',
+            height: '1024px',
+          },
+          type: 'desktop',
+        },
+      },
+    },
   },
+  decorators: [
+    withThemeByClassName<ReactRenderer>({
+      themes: {
+        light: '',
+        dark: 'dark',
+      },
+      defaultTheme: 'light',
```

Comment:
Why light theme's value is empty and not `light`?

Reply:
A good point. I add it.